notebooks/official/workbench/demand_forecasting/forecasting-retail-demand.ipynb

```diff
@@ -75,7 +75,7 @@
         "id": "dbab58d4ae1a"
```

Comment:
<p>combine 2 text cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1200/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>Cells which have <strong>#@bigquery</strong> (ex: #@bigquery CREATE SCHEMA [your-dataset-id] OPTIONS( location="us" )) will be bigquery magic cells in Vertex AI managed notebooks, so keeping them in a new cell</p>